bin/client.py

```diff
@@ -278,25 +278,42 @@ def main():
     install_exc_handler(default_handler)
     ver = 'APEL client %s.%s.%s' % __version__
     opt_parser = OptionParser(version=ver, description=__doc__)
+    DEFAULT_CONF_LOCATION = '/etc/apel/client.cfg'
+    DEFAULT_SSMCONF_LOCATION = '/etc/apel/sender.cfg'
+    DEFAULT_LOGCONF_LOCATION = '/etc/apel/logging.cfg'
 
     opt_parser.add_option('-c', '--config',
-                          help='main configuration file for APEL',
-                          default='/etc/apel/client.cfg')
+                          help=('main configuration file for APEL, '
+                                'default path: ' + DEFAULT_CONF_LOCATION),
+                          default=DEFAULT_CONF_LOCATION)
 
     opt_parser.add_option('-s', '--ssm_config',
-                          help='location of SSM config file',
-                          default='/etc/apel/sender.cfg')
+                          help=('location of SSM config file, '
+                                'default path: ' + DEFAULT_SSMCONF_LOCATION),
+                          default=DEFAULT_SSMCONF_LOCATION)
 
     opt_parser.add_option('-l', '--log_config',
-                          help='location of logging config file (optional)',
-                          default='/etc/apel/logging.cfg')
+                          help=('location of logging config file (optional), '
+                                'default path: ' + DEFAULT_LOGCONF_LOCATION),
+                          default=DEFAULT_LOGCONF_LOCATION)
 
     options, unused_args = opt_parser.parse_args()
-    ccp = ConfigParser.ConfigParser()
-    ccp.read(options.config)
 
-    scp = ConfigParser.ConfigParser()
-    scp.read(options.ssm_config)
+    # check if config file exists using os.path.isfile fuction
+    if os.path.isfile(options.config):
```

Comment:
Could you refactor this into a function and then just call it with `config` and `ssm_config` (to replace the check at line here: https://github.com/apel/apel/pull/299/files#diff-3a1e2d7257907d68bb0bd07963020f8d03268ea7a0d680abebf80278327718d4R311 )?

Reply:
done